src/Layouts/LayersList/LayerListBox.vala

```diff
@@ -323,6 +323,7 @@ public class Akira.Layouts.LayersList.LayerListBox : VirtualizingListBox {
         var node1 = im.node_from_id (layer1.id);
         var node2 = im.node_from_id (layer2.id);
 
+        // Why does this crash?
         var node1_is_group = node1.instance.is_group;
         var node2_is_group = node2.instance.is_group;
```

Comment:
@mbfraga This creates a `SEGFAULT` and I can't figure out why

Reply:
Uh, actually it seems that `node1.instance` doesn't exist.
I'm definitely doing something wrong.

Reply:
All right, I think I figured what I was doing wrong, which was removing the original node before the cloned instance had the chance to be used to create the new node.